### ISSUE TITLE
[BUG] Traces duration are gone after upgrading from 0.41.0

### ISSUE THREAD

**Issue by dcsg** [DataDog/dd-trace-php#855]
**Describe the bug**

I've noticed that after upgrading from 0.41.0 to 0.44.0 the traces lost the execution time. I've attached a screenshot of it.

![image](https://user-images.githubusercontent.com/744921/80579561-2eebe280-8a02-11ea-8f79-dbd2241a4255.png)

**Comment by labbati** [comment]
Hi @dcsg, thanks for reporting this. We believe we have identified the issue. Do you have a dev/staging/testing env when you can confirm this rc https://146286-119990860-gh.circle-artifacts.com/0/datadog-php-tracer_1.0.0-nightly_amd64.deb fixes the issue for you?

**Comment by dcsg** [comment]
do you have the `.x86_64.rpm` version of it?

**Comment by labbati** [comment]
here it is: https://146286-119990860-gh.circle-artifacts.com/0/datadog-php-tracer-1.0.0_nightly-1.x86_64.rpm

**Comment by labbati** [comment]
We have released [0.44.1](https://github.com/DataDog/dd-trace-php/releases/tag/0.44.1) which reverts the changes to the clock we use internally that does not work in specific circumstances. 

It would be great you can confirm this version fixes the issue you are experiencing.

We are still continuing to investigate to understand why this issue was not detected by our CI and to add proper tests to make sure this will not happen again in the future. We will provide updates in this issue once the plan is clear.

**Comment by dcsg** [comment]
I've deployed to staging and it still continues and also the flame graph is not ok

**Comment by labbati** [comment]
could you please create a support ticket, as we need at this point to look into your account and see if this can be related to something else or a sum of different factors?

**Comment by EtienneRd** [comment]
I also have the issue with 0.44.0, didn't test 0.44.1

Going back to 0.41.0 fixed it for me

**Comment by morrisonlevi** [comment]
@EtienneRd Version 0.44.1 fixes a bug that is present in 0.43.0 and 0.44.0 where the durations are always zero on some configurations of PHP (it's dependent on how PHP was configured). It seems this may not be the same issue that dcsg is hitting, but it would be good to know if v0.44.1 fixes your issue or not.

**Comment by SammyK** [comment]
Hey @dcsg! I'm just following up after @labbati's request. Were you able to [create a support ticket](https://docs.datadoghq.com/help/)? We can dig more deeply into the issue once that is created. Also, I just wanted to confirm that you're upgrading using [this rpm package](https://github.com/DataDog/dd-trace-php/releases/download/0.44.1/datadog-php-tracer-0.44.1-1.x86_64.rpm), correct?

**Comment by dcsg** [comment]
@SammyK yes, it is the 0.44.1 that is in staging. I will open the ticket today then.

**Comment by dcsg** [comment]
ticket id #336524

**Comment by EtienneRd** [comment]
@morrisonlevi 0.44.1 fixes the 0 durations, thanks

**Comment by dcsg** [comment]
@SammyK @morrisonlevi I deployed it again and it seems to be fixed.

**Comment by SammyK** [comment]
@EtienneRd @dcsg Thank you so much for confirm that this fixed the issue you for. I'm very sorry for the hassle. I really appreciate you taking the time to report and follow up on this issue.

I'm going to close this issue for now, but feel free to reopen if you have any more trouble.

**Comment by SammyK** [comment]
Hey @dcsg! If you don't mind me asking. Was there anything that you changed between the first attempt that that patch release was applied verses the second attempt that worked?

**Comment by dcsg** [comment]
I guess it was a bad deploy or that the agent might not have rebooted properly. To be honest I didn't look much in depth to understand why it didn't worked well at first.

**Comment by SammyK** [comment]
Gotcha. Thanks for the following up!

**Comment by sprymiker** [comment]
The fix helps. Confirmed.
